python/cudf/cudf/core/dataframe.py

```diff
@@ -714,7 +708,10 @@ def __repr__(self):
         # columns because we do not support NaT - pyarrow as_column
         # preprocessing converts NaT input values from numpy or pandas into
         # null.
-        output = output.to_pandas().__repr__().replace(" NaT", "null")
+        output = output.to_pandas().__repr__()
+        if ' NaT' in output:
+            output = output.replace(" NaT", "null")
```

Comment:
Did you find this to be more performant? Curious as to why you made this change.